app/queue.py

```diff
@@ -50,7 +50,7 @@ def inflight_prefix(self, suffix: str = None, process_type: str = None) -> str:
             return f"{Buffer.IN_FLIGHT.value}::{str(process_type)}"
         return f"{Buffer.IN_FLIGHT.value}"
 
-    def inflight_name(self, receipt: UUID = uuid4(), suffix: str = None, process_type: str = None) -> str:
+    def inflight_name(self, receipt: UUID = uuid4(), suffix: str | None = None, process_type: str | None = None) -> str:
```

Comment:
```suggestion
    def inflight_name(self, receipt: UUID = uuid4(), suffix: Optional[str] = None, process_type: Optional[str] = None) -> str:
```

Reply:
Thanks Steve!  Is `Optional[]` the preferred syntax?  If so, I'll change them!

Reply:
Yeah, according to the docs `Optional` is preferred: https://mypy.readthedocs.io/en/stable/kinds_of_types.html#optional-types-and-the-none-type